src/mongo/config.h.in

```diff
@@ -119,17 +119,5 @@
 
 // Percona specific section
```

Comment:
I'm wondering if we can remove this comment too. There is just the `mongo_config_have_basic_stringbuf_str_rvalue` below added by us but this is not 'Percona specific'.